### PR TITLE
Completed es6-classes exercise.

### PR DESCRIPTION
**Pull request by sharon-tieu**
Screen Shot 2022-10-25 at 9 36 30 AM
Screen Shot 2022-10-25 at 9 36 50 AM

### CODE REVIEW

File: es6-classes/example-class.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function ExampleClass() {
+
+}
```
Comment (scbowler) [inline es6-classes/example-class.js]: The `ExampleClass` should be a `class` 
```JAVASCRIPT
class ExampleClass {}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

**Comment by sharon-tieu** [comment]
Screen Shot 2022-10-25 at 9 43 34 AM

File: es6-classes/example-class.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class ExampleClass {
+  constructor(name) {
+    this.name = name;
+  }
```
Comment (scbowler) [inline es6-classes/example-class.js]: It didn't need anything inside of it, it could have been left empty.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍 

1 FYI comment, no changes required

File: es6-classes/example-class.js (hunk 2 above)
Comment (sharon-tieu) [reply es6-classes/example-class.js]: Ok. I see. So `exampleClass` is changed to a `class` but the code block can be left empty? Such as:

> class ExampleClass {

> }

**Pull request merged**